CMakeLists.txt

```diff
@@ -143,6 +143,7 @@ find_package(intx CONFIG REQUIRED)
 add_library(evmone evmone/lib/evmone/analysis.cpp
```

Comment:
I should handle this integration next. Are you still using Hunter to other dependencies?

Reply:
@chfast Yes, still Hunter